src/lib/box.js

```diff
@@ -698,6 +698,15 @@ foam.CLASS({
     'foam.dao.BoxDAOListener'
   ],
   methods: [
+    function put_(x, obj) {
+      return this.SUPER(this.Serializable.isInstance(x) ? x : null, obj);
```

Comment:
I think we can just pass null, since we would never want to serialize the context.